languages/python/polar/build.py

```diff
@@ -15,7 +15,14 @@
     "CI": "native",
 }
 env = os.environ.get("ENV", "DEVELOPMENT")
-lib = lib_dirs[env] + "/libpolar.a"  # @TODO: Windows
+libs = []
+if sys.platform.startswith("win"):
+    libs.append(lib_dirs[env] + "/polar.lib")
+    libs.append("Ws2_32.lib")
```

Comment:
How did you find out this is what's needed?

Reply:
googling unknown symbol link errors one at a time to see what lib they're in

Reply:
Nice